src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/enable-lambda-monitoring/containerized-images.mdx

```diff
@@ -0,0 +1,75 @@
+---
+title: Instrument your containerized image layer
+metaDescription: A guide for instrumenting your containerized image layer with New Relic.
+freshnessValidatedDate: never
+---
+
+If you're using a containerized image for Lambda, you'll want to add New Relic to your code. After you add our pre-built layer to your image, you'll need to push the image to your AWS Elastic Container Registry. From there, whenever you run your function, New Relic will run, too. 
```

Comment:
I like this intro, but it feels like it's missing a CTA. I think if there's a reason explaining why I want to add New Relic to my code, that would really clinch it.

Also, I think the ", too" should probably be swapped to something that doesn't require a comma since there are 3 of them in the final sentence.

Reply:
Fixed!